build_shared.sh

```diff
@@ -13,6 +13,14 @@ echo "ANDROID_ROOT: ${ANDROID_ROOT}"
 echo "KERNEL_TOP  : ${KERNEL_TOP}"
 echo "KERNEL_TMP  : ${KERNEL_TMP}"
 
+if [ "$use_system_clang" = true ] ; then
+    CROSS_COMPILE="aarch64-linux-gnu-"
+    CROSS_COMPILE_ARM32="arm-none-eabi-"
+else
+    CROSS_COMPILE="aarch64-linux-android-"
+    CROSS_COMPILE_ARM32="arm-linux-androideabi-"
+fi
```

Comment:
That's for system gcc?

Reply:
that's what was there for (presumably) android clang

Reply:
Fwiw these are not present in the given `CLANG_A13` path so I really wouldn't be surprised if they're now unused since 9ff78e08c39000fbc0f9868c1eea844d891ab6d9?